tools/serverpod_cli/lib/src/logger/loggers/std_out_logger.dart

```diff
@@ -1,16 +1,21 @@
 import 'dart:io';
 import 'dart:math' as math;
 
-import 'package:colorize/colorize.dart';
 import 'package:serverpod_cli/src/analyzer/code_analysis_collector.dart';
 import 'package:serverpod_cli/src/logger/logger.dart';
+import 'package:serverpod_cli/src/util/ansi_style.dart';
+import 'package:serverpod_cli/src/logger/helpers/progress.dart';
 import 'package:source_span/source_span.dart';
 import 'package:super_string/super_string.dart';
 
 /// Logger that logs using the [Stdout] library.
 /// Errors and Warnings are printed on [stderr] and other messages are logged
 /// on [stdout].
 class StdOutLogger extends Logger {
+  static const int _defaultColumnWrap = 100;
```

Comment:
Perhaps this should be 80 to match the default width of Dart?

Reply:
The value 100 was inspired by Flutters default value for a box.

But sounds good to base it on the default value of Dart.